### PR TITLE
upgrade ui-bootstrap to 0.14.3 

### LINKED ISSUE #2321: [enhancement] update ui-bootstrap
ui-bootstrap latest version has some breaking changes we need to update accordingly

### PR DESCRIPTION
**Pull request by sendilkumarn**
changed modal into uibModal
modalInstance into uibModalInstance
progressbar into uib-progressbar
alert into uibAlert

fix #2321

### CODE REVIEW

**Comment by mraible** [comment]
👍(I did not checkout the branch and try it, just a code review)

File: entity/templates/src/main/webapp/app/_entity-controller.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 angular.module('<%=angularAppName%>')
-    .controller('<%= entityClass %>Controller', function ($scope, $state, $modal<% if (fieldsContainBlob) { %>, DataUtils<% } %>, <%= entityClass %><% if (searchEngine == 'elasticsearch') { %>, <%= entityClass %>Search<% } %><% if (pagination != 'no') { %>, ParseLinks<% } %>) {
-      
+    .controller('<%= entityClass %>Controller', function ($scope, $state, $uibModal<% if (fieldsContainBlob) { %>, DataUtils<% } %>, <%= entityClass %><% if (searchEngine == 'elasticsearch') { %>, <%= entityClass %>Search<% } %><% if (pagination != 'no') { %>, ParseLinks<% } %>) {
```
Comment (deepu105) [inline entity/templates/src/main/webapp/app/_entity-controller.js]: I guess we can remove this injection as it is not used in the controller

**Comment by jdubois** [comment]
@sendilkumarn there's a merge conflict, can you have a look? I don't think it's a complicated one

**Comment by sendilkumarn** [comment]
@jdubois  resolved the conflict.

**Pull request merged**